espresso/docker-compose.yml

```diff
@@ -1,39 +1,258 @@
 # Espresso OP Integration Docker Setup
 
 services:
+  l1-genesis:
+    restart: on-failure
+    build:
+      context: ../
+      dockerfile: espresso/docker/op-stack/Dockerfile
+      target: op-deployer-target
+    image: op-espresso-deployment-utils:espresso
+    volumes:
+      - ./deployment/l1-config:/config
+      - ./docker/l1-geth:/templates:ro
+      - l1-data:/data
+    entrypoint: ["/bin/bash", "-c"]
+    command:
+      - |
+        set -euo pipefail
+
+        echo "Updating genesis timestamp..."
+        dasel put -f /config/genesis.json -s .timestamp -v $(printf '0x%x\n' $(date +%s))
+
+        echo "Generating consensus layer genesis..."
+        eth-beacon-genesis devnet \
+                          --quiet \
+                          --eth1-config "/config/genesis.json" \
+                          --config "/templates/beacon-config.yaml" \
+                          --mnemonics "/templates/mnemonics.yaml" \
+                          --state-output "/config/genesis.ssz"
+        cp -r /templates/beacon-config.yaml /config/config.yaml
+
+        echo "Generating validator keys..."
+        rm -rf /config/keystore && \
+        eth2-val-tools keystores --out-loc /config/keystore \
+                                 --source-mnemonic "$(yq -r '.[0].mnemonic' "/templates/mnemonics.yaml")" \
+                                 --source-min 0 \
+                                 --source-max 1
+        mkdir -p /data/lighthouse-validator
+        mkdir -p /data/lighthouse-validator/validators
+        cp -r /config/keystore/keys/* /data/lighthouse-validator/validators/
+        cp -r /config/keystore/secrets/ /data/lighthouse-validator/
+
+        if [[ ! -f "/config/jwt.txt" ]]; then
+          echo "Generating JWT secret..."
+          openssl rand -hex 32 > "/config/jwt.txt"
+        fi
+
+  l1-validator:
+    image: sigp/lighthouse
+    depends_on:
+      l1-genesis:
+        condition: service_completed_successfully
+      l1-geth:
+        condition: service_started
+      l1-beacon:
+        condition: service_started
+    volumes:
+      - l1-data:/data
+      - ./deployment/l1-config:/config:ro
+    command:
+      - lighthouse
+      - validator_client
+      - --testnet-dir
+      - /config
+      - --beacon-nodes
+      - http://l1-beacon:${L1_BEACON_PORT}
+      - --init-slashing-protection
+      - --datadir
+      - /data/lighthouse-validator
+      - --suggested-fee-recipient
+      - ${OPERATOR_ADDRESS}
+
+  l1-beacon:
```

Comment:
`l1-beacon` fails to run with the following error:
```
Unable to open testnet dir at "/config": Unable to open deposit_contract_block.txt: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

But `deposit_contract_block` doesn't seem to be referenced anywhere in the codebase. Is it a file you added locally but didn't commit?

Reply:
Yes, missed that one
Fixed